dataland-frontend/tests/e2e/specs/eu-taxonomy-financials/AddRemoveKpis.ts

```diff
@@ -36,46 +32,10 @@ describeIf(
         DataTypeEnum.EutaxonomyFinancials,
         testCompany,
         testData.t,
-        fillAndValidateEuTaxonomyForFinancialsUploadForm,
+        (data) => fillAndValidateEuTaxonomyCreditInstitutionForm(data),
```

Comment:
This can be reverted to main, because you don't need a form-filling-function anymore for your new test, if you just upload the stuff via the api there as we discussed today.

Reply:
This test needs the form to be filled out as it tests adding and removing of KPIs.. manually.

Reply:
I only meant this change,  but at second glance it is Clearer better now.   Will resolve it